process/block/metablock.go

```diff
@@ -598,6 +598,9 @@ func (mp *metaProcessor) saveMetricCrossCheckBlockHeight() {
 
 		crossCheckBlockHeight += fmt.Sprintf("%d: %d", i, valueStored)
```

Comment:
Better:
remove 595-597 lines
replace this with:
```GO
crossCheckBlockHeight += fmt.Sprintf("%d: %d, ", i, valueStored)
```

Reply:
done